vms/secp256k1fx/camino_keychain.go

```diff
@@ -131,13 +125,31 @@ func TraverseOwners(out *OutputOwners, msig AliasGetter, callback TraverserOwner
 				if !ok {
 					return 0, errWrongOwnerType
 				}
-				stack = append(stack, &stackItem{owners: owners})
-				goto Stack
+				success, err := callback(
```

Comment:
in which case we expect success to be true?

Reply:
If the cb returns true, it is meant as "verified" and no fürther traversal is required.
This can only be the case for VerifySpend (for builder) all other implementations return false always.